### PR TITLE
COMMERCE-7657 removed date check for order item request delivery date

### PR DESCRIPTION
**Pull request by andrea-ale-sbarra**
https://issues.liferay.com/browse/COMMERCE-7657

### CODE REVIEW

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

**Comment by andrea-ale-sbarra** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemLocalServiceImpl.java [hunk 1]
```diff
@@ -619,12 +618,6 @@ public CommerceOrderItem updateCommerceOrderItemDeliveryDate(
 			long commerceOrderItemId, Date requestedDeliveryDate)
 		throws PortalException {
 
-		if ((requestedDeliveryDate != null) &&
-			requestedDeliveryDate.before(new Date())) {
-
-			throw new CommerceOrderItemRequestedDeliveryDateException();
```
Comment (luca-pellizzon) [inline modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CommerceOrderItemLocalServiceImpl.java:625]: This is done for both imported items and "normal" orders. Does it mean that we can add a delivery date in the past even for current orders?
If this is true we should create a specific method for import only usage.

**Review by luca-pellizzon — APPROVED** [review]

**Comment by luca-pellizzon** [comment]
ci:forward

**Pull request closed without merging**